.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Question
+labels: question
+about: Ask a general question related to OpenSSL development
+---
+
+<!--
+Thank you for your interest in OpenSSL. If this is your first question,
+please take the time to read the following lines before posting it.
+
+NOTE:
+
+    If you're asking about how to use OpenSSL, this isn't the right 
+    forum.  Please see our User Support resources:
+    https://github.com/openssl/openssl/blob/master/.github/SUPPORT.md
+
+For questions related to build issues:
+
+    Please use the 'Bug report' template.
+
+For other questions:
+
+    Please describe your problem as concise as possible while giving
+    us enough information to understand your problem. An example code
+    or example commands are highly appreciated if they help us to
+    better understand what you are trying to achieve.
+
+    Also, please remember to tell us which OpenSSL version you are
+    using and whether it is system provided or you built it yourself.
+    In the latter case, please also send us your build configuration.
+    With OpenSSL before 1.1.1, the configuration output comes from the
+    configuration command.  With OpenSSL 1.1.1 and on, it's the output
+    of `perl configdata.pm --dump`
```

Comment:
Perhaps specify where the command needs to be run from.

Reply:
Addressed in 0c4dbb5cdc49f1213d66775c8ebe9e0aeba67da8.